EDI/UserInterface/01a_Exports.md

```diff
@@ -0,0 +1,340 @@
+
+- ![Delete](../../Assets/Icons/Trash03.png "[Delete]") (Delete)   
+    Click this button to delete the selected exporter definition, see [Delete definition](../Operation/01_ManageDefinitions.md#delete-definition). This button is only displayed if a single checkbox in the list of exporter definitions is selected.
+   
+
+The following functions and fields are available in this view:
+
+- *Definition name*  
+    Name of the exporter definition.
+
+- *Status*  
+    Exporter definition status. The following statuses are available:
+    - ![Status](../../Assets/Icons/Status01.png "[Status]") **Active**   
+        The exporter definition is active and can be used to send messages. 
+    - ![Status](../../Assets/Icons/Status04.png "[Status]") **Inactive**   
+        The exporter definition is inactive and cannot be used to send messages.   
+
+- *Attribute set*  
+    Attribute set of the exporter definition that contains the attributes of the EDIFACT message.
+
+- *Access key*  
+    Access key of the exporter definition. It is a key that you can give to your business partner to access your *Actindo* tenant without logging in. You can freely specify any string.
+    
+- *Download link*   
+   Preview of the message fields. When you click on this link, a Browser window is opened which contains the already defined message fields in a string, see [Preview definition](../Operation/01_ManageDefinitions.md#preview-definition).
+
+- *Export link*   
+<!---wofür wird der Export link benötigt?-->
+   
+- ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add)  
+    Click this button to add an exporter definition. The *Create definition* view is displayed, see [Create exporter definition](#create-exporter-definition).
+
+
+## Create exporter definition
+
+*Exports > Tab EXPORTS > Button Add*
+
+![Create definition](../../Assets/Screenshots/EDI/Operation/ExportDefinitionCreate.png "[Create definition]")
+
+**Create definition**
+
+The *Create definition* view includes the *Settings* tab which contains the fields and functions needed to define a basic definition, see [Settings](#settings-tab).
+
+- *Label*   
+   Enter the name of the exporter definition.
+
+- ![Active](../../Assets/Icons/Toggle.png "[Active]") (Active)   
+   Enable this toggle to activate the exporter definition after you have finished configuring it. Disable this toggle if you no longer want to use the exporter definition.
+
+- *Template engine*   
+   Use this drop-down list to select the engine with which *Actindo* shall process the message. For EDIFACT messages, the **XmlExportTemplateEngine** is used. This field is read-only after you have saved a newly created definition.
```

Comment:
Click the drop-down list to select...

Reply:
done, ne das spielt hier ja gar keine Rolle